llama_stack/core/server/quota.py

```diff
@@ -15,7 +15,7 @@
 from llama_stack.providers.utils.kvstore.config import KVStoreConfig, SqliteKVStoreConfig
 from llama_stack.providers.utils.kvstore.kvstore import kvstore_impl
 
-logger = get_logger(name=__name__, category="quota")
+logger = get_logger(name=__name__, category="core")
```

Comment:
same here, should we make a new category?

Reply:
👍🏽